src/Build.UnitTests/Graph/GraphLoadedFromSolution_tests.cs

```diff
@@ -724,19 +725,9 @@ private static bool EdgeCompliesWithSolutionDependency((ConfigurationMetadata, C
 
             foreach (var node in graphFromSolution.ProjectNodes)
             {
-                // Project references get duplicated, once as entry points from the solution (handled in the if block) and once as nodes
```

Comment:
This comment was all wrong. Because of `AssignProjectConfigurations`, only the configuration/platform defined in the sln should be used so that each project builds exactly once.

Reply:
Seems like a reasonable assertion to add